### DISCUSSION TITLE
🚀 **Loco v0.11.0 is out!** 🔥

### DISCUSSION THREAD

**Discussion by jondot** [loco-rs/loco#863] [Announcements]
🚀 **Loco v0.11.0 is out!** 🔥

We’re excited to announce a major release packed with features, improvements, and fixes that will supercharge your Loco experience!

✨ **What’s new?**
- **SeaORM upgrade**: Now at v1.1.0! Seamless integration with the latest and greatest.
- **OpenAPI example**: Added for better API documentation and exploration.
- **Health route improvements**: Keeping things lean and mean! [#851](https://github.com/loco-rs/loco/pull/851)
- **Sqlite pragmas**: Optimized performance for SQLite users. [#848](https://github.com/loco-rs/loco/pull/848)
- **Rsbuild upgrade**: Now on v1.0, bringing more stability to your builds. [#792](https://github.com/loco-rs/loco/pull/792)
- **Debug implemented**: Added `fmt::Debug` to public structs for clearer output. [#812](https://github.com/loco-rs/loco/pull/812)
- **Sidekiq queue config**: Control workers with the new `num_workers` setting. [#823](https://github.com/loco-rs/loco/pull/823)
- **Starter code tweaks**: Cleaned up comments for clarity and accuracy. [#824](https://github.com/loco-rs/loco/pull/824)
- **Y2038 bug fix**: Say goodbye to the JWT issue on 32-bit platforms. [#825](https://github.com/loco-rs/loco/pull/825)
- **Clickable App URL**: The Boot Banner just got more interactive! [#826](https://github.com/loco-rs/loco/pull/826)
- **Banner control**: Don’t want to see the banner? Use the `--no-banner` flag. [#839](https://github.com/loco-rs/loco/pull/839)
- **on_shutdown hook**: New lifecycle hook added for graceful shutdown handling. [#842](https://github.com/loco-rs/loco/pull/842)

Get the latest version of Loco now and start building faster, stronger, and smarter!

Thanks to our senior and new committers! 

@Sillyvan @felipesere @vishnuMohanan01 @balcsida @omninonsense @jacovdbergh @mrothNET @isaacdonaldson

**Comment by leonardodepaula** [comment]
Congratulations for this achievement. I don't have a rails background, but I have used Django for the past few years and a batteries included framework is very welcome.

**Comment by bendo01** [comment]
thank you for this great framework, coming from laravel, still noob on rust, but i found many similarity between loco and laravel
may i ask where is OpenAPI example ? trying to find on example directory

**Comment by zhuravkov** [comment]
I also did not find an example with OpenAPI. This is a necessary feature!
